torchgeo/datasets/utils.py

```diff
@@ -45,6 +46,8 @@
     "percentile_normalization",
 )
 
+Path: TypeAlias = Union[str, bytes, os.PathLike[bytes], os.PathLike[str]]
```

Comment:
```suggestion
Path: TypeAlias = Union[str, bytes, os.PathLike[str], os.PathLike[bytes]]
```
Just to keep the order consistent